### PR TITLE
Gists support, new way of loggers specification

### PR DESCRIPTION
**Pull request by weirdGuy**
Added support for loggin into gists(#17), also added "room" parameter to gdoc logging(#22).
Edited config according to specification #8

### CODE REVIEW

**Comment by rufuspollock** [comment]
Looks reasonable but I don't see *any* tests so can't approve this PR. The PR **must** include tests for the new features or a detailed explanation of why tests are not possible or not needed.

Small comment re commit messages: saying "Update main.coffee" does not tell me a lot. Please have informative commit messages that tell the reader (that includes you in the future) something about what is really happening.

**Comment by weirdGuy** [comment]
@rufuspollock Commit messages about "update main.coffee" was made through github and include only deletion of commented code that not needed

**Comment by rufuspollock** [comment]
@weirdGuy 

> Commit messages about "update main.coffee" was made through github and include only deletion of commented code that not needed

then say that in the commit message e.g.

```
[xs]: Delete commented code from main.coffee that is not needed.
```

**Comment by weirdGuy** [comment]
@rufuspollock alright, so i need to revert commits? and push them again?

File: test/messages-test.js [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
     })
 
+    it('Gist messaging', function () {
+        let message = {
+            "text": "+todo testing @test",
+            "user": {
+                "name": "test"
+            }
+        }
+        sinon.stub(messages, "sendGist", function () {
```
Comment (rufuspollock) [inline test/messages-test.js:32]: You need to stub at a deeper level than this. As it currently stands this test is pointless as it does not test anything.

You would really want to stub the method that does the call out to the github gist API if you could.

File: test/messages-test.js [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
-                        text: "+todo @assignees some text",
-                        user: {
-                            name: "Artem (weirdGuy)",
-                            login: "weirdGuy"
-                        }
-                    }))
-            },
-        ])
+        }
+        sinon.stub(messages, "sendMessage", function () {
```
Comment (rufuspollock) [inline test/messages-test.js:15]: You need to stub at a deeper level than this - as it currently stands this test is pointless (it does not test anything).

You want to stub method that does the call out to the google docs API if you could e.g. stub the doc.addRow method in the underlying method.

**Comment by rufuspollock** [comment]
@weirdGuy no, don't bother but this is important for the future.

**Review by rufuspollock — CHANGES_REQUESTED** [review]
See comments.

**Comment by rufuspollock** [comment]
I'm going to merge this for now even though tests are empty so we can keep moving forward (and deploying)!

**Pull request merged**